src/LocationPickerWidget.css

```diff
@@ -1,3 +1,499 @@
+.map-container {
```

Comment:
since we're starting to have decently large stylesheets we should probably make an issue to move to sass or something like it. Mostly so we can start making vars for colors and sizes etc.